config/locales/en.yml

```diff
@@ -2266,7 +2266,7 @@ en:
       welcomemat:
         url: https://welcome.openstreetmap.org/
         title: For Organizations
-        description: With an organization making plans for OpenStreetMap? Find what you need to know in the Welcome Mat.
+        description: Do you belong to an organization that wants to work with OpenStreetMap? Find what you need to know in the Welcome Mat.
```

Comment:
The old wording might read better in some other dialect of English, but it never felt grammatical to me because it starts with a preposition phrase and lacks a subject. The new wording more closely matches the masthead on the Welcome Mat site itself.